mbtiles/src/bin/mbtiles.rs

```diff
@@ -48,6 +48,15 @@ enum Commands {
         /// Value to set, or nothing if the key should be deleted.
         value: Option<String>,
     },
+    #[command(name = "diff")]
+    Diff{
+        /// MBTiles file to read from
+        src_file: PathBuf,
```

Comment:
I think a better pattern is `diff  FILE_A.mbtiles  FILE_B.mbtiles  DIFF.mbtiles`, and the explanation would be that diff will contain all changes going from A to B, i.e. all changes that must be made to A so it becomes B.